test/multiarray/CMakeLists.txt

```diff
@@ -1,42 +1,59 @@
-IF(JPEG_FOUND)
-  ADD_DEFINITIONS(-DHasJPEG)
-ENDIF(JPEG_FOUND)
-
-IF(PNG_FOUND)
-  ADD_DEFINITIONS(-DHasPNG)
-ENDIF(PNG_FOUND)
-
-IF(TIFF_FOUND)
-  ADD_DEFINITIONS(-DHasTIFF)
-ENDIF(TIFF_FOUND)
-
-VIGRA_ADD_TEST(test_multiarray test.cxx LIBRARIES vigraimpex)
-
-FILE(MAKE_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/impex)
-
-VIGRA_CONFIGURE_THREADING()
+# Check cpp version
+if(NOT ${VIGRA_CPP_VERSION})
+    message(FATAL_ERROR 
+            "cmake error: VIGRA_CPP_VERSION not defined yet. "
+            "Call VIGRA_DETECT_CPP_VERSION() from the main CMakeLists file." )
+endif()
 
-if(NOT THREADING_FOUND)
-    MESSAGE(STATUS "** WARNING: Your compiler does not support C++ threading.")
-    MESSAGE(STATUS "**          test_multiarray_chunked will not be executed on this platform.")
-    if(NOT WITH_BOOST_THREAD)
-        MESSAGE(STATUS "**          Try to run cmake with '-DWITH_BOOST_THREAD=1' to use boost threading.")
-    endif()
-    
+# Multiarray source files use 'auto' and other features from c++11.
```

Comment:
OK, The comment is not quite right.  The uses of `auto` I was referring to are in `test/multiarray/test.cxx`:

```
$ grep -r auto test/multiarray/
test/multiarray/CMakeLists.txt:# Multiarray source files use 'auto' and other features from c++11.
test/multiarray/test.cxx:        auto i3_f = createHierarchicalIterator(a3),
test/multiarray/test.cxx:        auto i3_c = zip(i3_f, i3_f);
test/multiarray/test.cxx:            auto i2_f = i3_f.begin ();
test/multiarray/test.cxx:            auto i2_c = i3_c.begin ();
test/multiarray/test.cxx:            auto i2_l = i3_f.end ();
test/multiarray/test.cxx:                auto i1_f = i2_f.begin ();
test/multiarray/test.cxx:                auto i1_c = i2_c.begin ();
test/multiarray/test.cxx:                auto i1_l = i2_f.end ();
```